Source/CovPassApp/Source/Resources/Locale/en.lproj/Localizable.strings

```diff
@@ -11,226 +11,226 @@
 
 /********** onboarding **********/
 
-"start_onboarding_title" = "Willkommen in der CovPass-App";
+"start_onboarding_title" = "Welcome to the CovPass app";
 
-"start_onboarding_message" = "Weisen Sie EU Digitale COVID-Zertifikate ganz einfach mit dem Smartphone nach.";
+"start_onboarding_message" = "Verify EU digital COVID certificates easily with your smartphone.";
```

Comment:
I don't have a better suggestion but this sounds like you can scan the QR-Codes from someone else to verify that the certificate is valid (what the CovPassCheck App does).